### PR TITLE
Add dependencies

### PR DESCRIPTION
**Pull request by pmiettinen**
- PyTables

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```
Comment (mookerji) [inline .travis.yml]: .... so you can do this with `sudo`, but it'll run it on older infrastructure that will take a while. When you set `sudo: false`, it'll run it in faster Linux containers.

File: setup.py [hunk 2]
```diff
@@ -8,6 +8,17 @@
   pass
 
 try:
+  from subprocess import STDOUT, check_call
+  import os
+  check_call(['apt-get', 'install', '-y', 'libhdf5-serial-dev'])
```
Comment (mookerji) [inline setup.py]: This is Debian-specific, so if you may want to change it to include os x instead.

Comment (mookerji) [inline setup.py]: Actually, my thought actually is to have an installation set of dependencies for travis and keep those in `.travis.yml`, and a separately deps.sh script in the repo that someone would run locally. Peregrine does something like this: https://github.com/swift-nav/peregrine/

**Comment by mookerji** [comment]
Yeah, thanks for fixing this. Unless anyone has any objections, my recommendation is to have the travis installation be standalone without sudo, and having a deps.sh script that someone can run on both OS X and Linux to get setup (example: https://github.com/swift-nav/peregrine/blob/master/deps.sh#L59). Thanks!

**Comment by pmiettinen** [comment]
@mookerji Sure, I'll try to move the approach to that direction. Thanks for the comments and tips.

**Comment by pmiettinen** [comment]
@rge-exafore Coulde you test this on osx please?

**Comment by mookerji** [comment]
@pmiettinen I'll go ahead and merge and you can actually follow up with an OS X change if you'd like. Travis can actually run tests on OS X too! 

See: 
- https://github.com/guinslym/gitopen/blob/08ee20fffad924d4a28bae2fbd7dccdcece14749/.travis.yml#L4
- https://github.com/fitnr/tweetcal/blob/0496b771a5f883ab2ad5be1779a6bd34ada1f7d5/.travis.yml

**Pull request merged**